src/data.tsx

```diff
@@ -0,0 +1,94 @@
+export const data_local = {
+
+  "data": {
+
+    "annual_salary":{
+      "type": "Annual Salary", 
```

Comment:
FYI, python has a method called `type` 
We changed `"type":` to `"classification":` in our model.